Makefile.in

```diff
@@ -1728,6 +1728,7 @@ echo-list	= for x in $1; do $(ECHO) "$$x"; done
 
 .PHONY: all
 all: $(default_pdf_targets) ;
+	killall -q -SIGHUP mupdf || true
```

Comment:
I think that's a good alternative. You can put a rule like that in your Makefile.ini, actually, and just use it.

Perhaps this is best left as a documentation change? I would be happy to see it added to the project as a recipe, but am a little more hesitant to add it to the main makefile.